runtime/shared_common/shrinit.cpp

```diff
@@ -1523,6 +1527,34 @@ hookFindSharedClass(J9HookInterface** hookInterface, UDATA eventNum, void* voidD
 	Trc_SHR_INIT_hookFindSharedClass_exit(currentThread);
 }
 
+/**
+ * Notify the open/close state of jar/zip files so as to force a timestamp check.
+ *
+ * @param [in] hookInterface  Pointer to pointer to the hook interface structure
+ * @param [in] eventNum  Not used
+ * @param [in,out] voidData  Pointer to a J9VMZipOpenStateEvent struct
+ * @param [in] userData  Not used
+ */
+void
+hookZipOpenState(J9HookInterface** hookInterface, UDATA eventNum, void* voidData, void* userData)
```

Comment:
Please name this better, like updateClasspathOpenState()

Reply:
Already fixed.